CMakeLists.txt

```diff
@@ -68,6 +68,10 @@ option(GENERATE_TS "Regenerate translation source files" OFF)
 option(USE_EXTERNAL_SINGLEAPPLICATION "Use external QtSingleApplication library" OFF)
 option(USE_LAUNCHER_ABSOLUTE_PATH "Use absolute path for the desktop launcher" ON)
 option(USE_WAYLAND_CLIPBOARD "USE KF Gui Wayland Clipboard" OFF)
+option(DISABLE_UPDATE_CHECKER "Disable check for updates" OFF)
+if (DISABLE_UPDATE_CHECKER)
+  add_definitions(-DDISABLE_UPDATE_CHECKER="ON")
```

Comment:
This is not needed. You can reference USE_WAYLAND_CLIPBOARD

Reply:
But it add definition in here:
https://github.com/flameshot-org/flameshot/blob/57c8e8d893669cd83d8b279ea3ff6db50b16aa48/src/CMakeLists.txt#L208-L211
Do I need to change to this way?

Reply:
And I find `add_compile_definitions` is more recommend.